tests/mocks/mocks.go

```diff
@@ -0,0 +1,36 @@
+package mocks
```

Comment:
These mocks are used in both the health and net modules tests. They should be used in `miner_test.go`, but it creates a circular dependency that needs to be resolved. Can't wrap my head around that at the moment, so leaving these mocks duplicated there in `miner_test.go` for now. Will ticket it up.